### PR TITLE
Update oauth_for_integrations.md

### PR DESCRIPTION
**Pull request by eho1307**
Adding two additional clarifications

### Merge instructions

Please merge once approved

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### CODE REVIEW

File: content/en/developers/integrations/oauth_for_integrations.md
Comment (drichards-87) [inline content/en/developers/integrations/oauth_for_integrations.md:172]:
```suggestion
If you've already published your OAuth client, remember to use the client ID and the client secret you were given at submission. The client secret was displayed only once, so if you've lost it, contact [ecosystems@datadog.com][11] for assistance.
```

File: content/en/developers/integrations/oauth_for_integrations.md
Comment (drichards-87) [inline content/en/developers/integrations/oauth_for_integrations.md:229]:
```suggestion
Datadog does not support multi-tenanted applications where customers authorize using individual subdomains; instead, authorization is supported only through a single domain.
```

**Review by drichards-87 — CHANGES_REQUESTED** [review]
Left some minor feedback on the PR.

**Review by drichards-87 — APPROVED** [review]

**Comment by eho1307** [comment]
@drichards-87 could you help me merge this?

**Comment by drichards-87** [comment]
@eho1307 Yep, no worries. I'll take care of that for you now.

Also, as an FYI, we recently enabled merge queue on the `documentation` repo. After you get the required approvals, you can add a comment with `/merge` to a PR, and it will automatically merge the PR.

**Pull request merged**
